src/App.tsx

```diff
@@ -1,24 +1,67 @@
 import * as React from 'react';
-import logo from './logo.svg';
+
 import styles from './App.module.scss';
+import * as api from './api';
+import Login from './components/Login';
+
+interface Props {}
+
+interface State {
+  profile: null | {
```

Comment:
Nit: I know this is just for demo purposes, so maybe it would be overkill, but if this were production code I believe we'd want a `Profile` type defined.

Reply:
agreed